sdk/identity/azure-identity/src/main/java/com/azure/identity/AuthorizationCodeCredentialBuilder.java

```diff
@@ -76,10 +76,10 @@ public AuthorizationCodeCredentialBuilder redirectUrl(String redirectUrl) {
     }
 
     /**
-     * Sets the client secret for the authentication. This is required for AAD web apps. Do not set this for AAD native
-     * apps.
+     * Sets the client secret for the authentication. This is required for Microsoft Entra web apps.
+     * Do not set this for Microsoft Entra native apps.
```

Comment:
@g2vinay to confirm but I think this maybe needs a `<p>` to render the line break you want.